example/src/providers/auth-service/app-auth/ionicAuthorizationRequestHandler.ts

```diff
@@ -77,11 +77,11 @@ export class IonicAuthorizationRequestHandler extends AuthorizationRequestHandle
 
         let authRequestKey = await this.storageBackend.getItem(authorizationRequestKey(handle))
         let json = await JSON.parse(authRequestKey);
-        
+
         let request = await AuthorizationRequest.fromJson(json);
 
         let response = await this.storageBackend.getItem(AUTHORIZATION_RESPONSE_KEY);
-        let parts = response.split('#');
+        let parts = response.split('?');
```

Comment:
The identity server default is # have you changed this on your server and this is why you modified this?

Reply:
If I'm not mistaken (it's been a long time now), this part interprets the authorization response of the oidc server. The following RFC describes this response: https://tools.ietf.org/html/rfc6749#section-4.1.2. The response must be something like that : 
```
HTTP/1.1 302 Found
Location: https://client.example.com/cb?code=SplxlOBeZQQYbYS6WxSbIA&state=xyz
```
As you can see, it's a `?` and not a `#`. 
I used Identity Server 3 and it sent a `?`. Recently, I've migrated to Identity Server 4 and it works the same.

However, I'm not the one who set up Identity Server, so I may be unaware of a specific configuration somewhere...

Reply:
I noticed that the `#` is used with the Implicit grant according to the RFC (https://tools.ietf.org/html/rfc6749#section-4.2.2). The code of this commit handles the Authorization Code grant.